fixups/RegLegacyFixups/RegistryFixups.cpp

```diff
@@ -570,7 +570,12 @@ LSTATUS __stdcall RegOpenKeyExFixup(
     Log("[%d] RegOpenKeyEx:\n", RegLocalInstance);
 
 
-    std::string keypath = InterpretKeyPath(key) + "\\" + InterpretStringA(subKey);
+    std::string keypath = InterpretKeyPath(key);
+    if (keypath.size() && keypath.back() != '\\')
```

Comment:
Move this change inside InterpretKeyPath function.

Reply:
Moved.